packages/melos/lib/src/common/utils.dart

```diff
@@ -46,6 +46,7 @@ const filterOptionDependsOn = 'depends-on';
 const filterOptionNoDependsOn = 'no-depends-on';
 const filterOptionIncludeDependents = 'include-dependents';
 const filterOptionIncludeDependencies = 'include-dependencies';
+const filterOptionSdkPath = 'sdk-path';
```

Comment:
`filterOptionSdkPath` -> `globalOptionSdkPath`

Reply:
Renamed in 7a9a9151fb73d5a2ee70692d55bcdd73c8c868f4